stratum/hal/lib/common/yang_parse_tree_test.cc

```diff
@@ -109,15 +109,17 @@ class YangParseTreeTest : public ::testing::Test {
   }
 
   // A proxy for YangParseTree::AddSubtreeInterface().
-  void AddSubtreeInterface(const std::string& name) {
+  void AddSubtreeInterface(const std::string& name,
+                           SingletonPort* singleton_p) {
     absl::WriterMutexLock l(&parse_tree_.root_access_lock_);
-
-    // Add one singleton port.
     SingletonPort singleton;
-    singleton.set_name(name);
-    singleton.set_node(kInterface1NodeId);
-    singleton.set_id(kInterface1PortId);
-    singleton.set_speed_bps(kTwentyFiveGigBps);
+    SingletonPort* singleton_ptr = (singleton_p == nullptr)? &singleton \
```

Comment:
Add space before `?`
Also please break the line from `=`
```
SingletonPort* singleton_ptr =
        (singleton_p == nullptr) ? &singleton : singleton_p;
```

Reply:
I encounter a problem....So I delete the branch and redo works on a new branch. 
I've sent a new pull request.